migrate/20250130_kubernetes_vm_size.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    alter_table(:kubernetes_nodepool) do
+      add_column(:target_node_size, :text, collate: '"C"', null: false, default: "standard-2")
+      add_column(:target_node_storage_size_gib, :bigint, null: true)
```

Comment:
If clover always picks the lowest option and this column has values, I prefer to set null: false for consistency.